scripts/ci/tags.yaml

```diff
@@ -516,3 +516,23 @@ peripheral_802154:
     - nrfxlib/nrf_rpc/
     - zephyr/modules/hal_nordic/nrf_802154/
     - zephyr/soc/nordic/
+
+bootloader:
+  files:
+    - bootloader/mcuboot/
+    - modules/crypto/mbedtls/
+    - modules/crypto/oberon-psa-crypto/
+    - nrf/drivers/hw_cc3xx/
+    - nrf/lib/fprotect/
+    - nrf/lib/hw_unique_key/
+    - nrf/lib/identity_key/
+    - nrf/lib/modem_antenna/
+    - nrf/lib/nrf_modem_lib/
+    - nrf/samples/bootloader/
+    - nrf/subsys/bootloader/
+    - nrf/subsys/fw_info/
+    - nrf/subsys/nrf_security/
+    - nrf/tests/subsys/bootloader/
```

Comment:
```suggestion
    - nrf/tests/modules/mcuboot/
    - nrf/tests/subsys/bootloader/
    - nrf/tests/subsys/dfu/
    - nrf/tests/drivers/fprotect/
    - nrf/tests/drivers/flash/
    - nrf/samples/nrf5340/extxip_smp_svr/
```

Reply:
Not really.
Those depencies are related to tests/subsys/bootloader only.
So this is a list for this specific tests not for all bootloader tests and samples.
Thus adding other tests does not fit the idea.

Reply:
For suggested, I will propose to add separated tag e.g dfu, fprotect flash etc.
But this is different PR.